src/main/fc/fc_init.c

```diff
@@ -607,6 +608,10 @@ void init(void)
     cmsDisplayPortRegister(displayPortCrsfInit());
 #endif
 
+#if defined (USE_HOTT_TEXTMODE) && defined (USE_CMS)
+    cmsDisplayPortRegister(displayPortHottInit());
```

Comment:
Might want to register this outside of `fc/fc_init.c` and instead in the telemetry subsystem.  Don't want to register multiple display ports and cause unwanted cycling between them when people aren't using the protocol or have telemetry enabled.

Reply:
I am doing the same for CMS over CRSF.